app/api/dao/user_extension.py

```diff
@@ -21,17 +21,50 @@ def create_user_extension(data):
                 A dictionary containing "message" which indicates whether or not the user_exension was created successfully and "code" for the HTTP response code.
         """
 
-        user_id = data["user_id"]
-        is_organization_rep = data["is_organization_rep"]
+        user_id = AUTH_COOKIE["user_id"].value
         timezone = data["timezone"]
+        
+        additional_info_data = {
+            "phone": "",
+            "mobile": "",
+            "personal_website": ""
+        }
 
-        user_extension = UserExtensionModel(user_id, is_organization_rep, timezone)
+        existing_additional_info = UserExtensionModel.find_by_user_id(user_id)
+        if existing_additional_info:
+            return messages.ADDITIONAL_INFORMATION_OF_USER_ALREADY_EXIST, HTTPStatus.CONFLICT
+        user_extension = UserExtensionModel(user_id, timezone)
+        if "is_organization_rep" in data:
```

Comment:
Use try except KeyError rather than if else to initialize dict elements
https://www.google.com/amp/s/www.geeksforgeeks.org/try-except-vs-if-in-python/amp/
EAFP principle

Reply:
got it. Thanks for the tips. Learned something new here. 😁. Changed to try-except.